src/dolphin/gx/GX_unknown_001.h

```diff
@@ -8,11 +8,11 @@
 /* 003FDDC0 */ extern const GXRenderModeObj lbl_804011E0;
 /* 003FDD84 */ extern const GXRenderModeObj lbl_804011A4;
 /* 003FDD48 */ extern const GXRenderModeObj GXNtsc480IntDf;
-/* 0033E33C */ extern UNK_FUNC(GXSetScissor);
```

Comment:
Not a fan of this macro, I take it.

Reply:
Nah it's fine. This change is because the func isn't unk anymore.